server/src/libs/environment/types.ts

```diff
@@ -0,0 +1,37 @@
+export interface EnvConfig {
+  [name: string]: string | undefined
+}
+
+export interface Environment {
+  __DEV__: boolean
+  ENV: string
+  REGION:
+    | 'us-central1'
+    | 'us-east1'
+    | 'us-east4'
+    | 'europe-west1'
+    | 'europe-west2'
+    | 'asia-east1'
+    | 'asia-east2'
+    | 'asia-northeast1'
+    | 'asia-northeast2'
+    | 'us-west2'
+    | 'us-west3'
+    | 'us-west4'
+    | 'northamerica-northeast1'
+    | 'southamerica-east1'
+    | 'europe-west3'
+    | 'europe-west6'
+    | 'europe-central2'
+    | 'australia-southeast1'
+    | 'asia-south1'
+    | 'asia-southeast1'
+    | 'asia-southeast2'
+    | 'asia-northeast3'
```

Comment:
généralement, j'aime bien limiter les valeurs possibles avec les types

dans ce cas précis, on ne maitrise pas les valeurs possibles

je ne vois pas trop l'interet de lister tout les datacenters de GCP

si un datacenter ouvre (ou moins risqué, ferme), les valeurs possibles pourront changer

si on ne peut pas (je n'ai pas vérifier si c'est possible ou non) réutiliser un type provenant de Firebase, je suggère de changer :

```suggestion
    string
```

Reply:
C'est un point sur lequel j'ai hésité et ai tranché, cela évite des erreurs et permet l'autocompletion pendant le dev, et disparait en production, si on dev on peut rajouter une valeur. Le fichier étant dans un dot env, on ne saisi plus cette valeur en dur dans le code, donc pourquoi pas revenir sur un string, je regarde après le retour des ops.